lutris/runners/linux.py

```diff
@@ -145,12 +145,12 @@ def get_launch_config_command(self, gameplay_info, launch_config):
 
         return command
 
-    def get_command(self):
+    def get_command(self) -> list:
```

Comment:
I think the element type is important, and here we know it- this should be List[str].